Marquee.js

```diff
@@ -0,0 +1,17 @@
+marqueeStocks();
```

Comment:
never call a functiion before it's decleration. it only work because of someting called "hoisting" (google it!)